influxdb_store.go

```diff
@@ -569,8 +568,8 @@ func (in *InfluxDBStore) init(server *influxDBServer.Server) error {
 	in.server = server
 	// TODO: Upgrade to client v2, see: github.com/influxdata/influxdb/blob/master/client/v2/client.go
```

Comment:
Ah good point, I missed that. Removed in ccf0b7a53014bfa4d1deb169b4adff326aae90e6